sklearn/preprocessing/imputation.py

```diff
@@ -102,11 +103,22 @@ class Imputer(BaseEstimator, TransformerMixin):
         - If `axis=0` and X is encoded as a CSR matrix;
         - If `axis=1` and X is encoded as a CSC matrix.
 
+    add_missing_indicator : boolean, optional (default=False)
+        If True, the transformed X will have an appended indicator matrix
+        to denote the features that have at least one missing value and
+        the indicator matrix has 1 for each missing value.
```

Comment:
I don't think they're really handled differently. I think "appended" is the only confusing thing here. Saying "new features are created" is unambiguous.